release_dashboard/lib/logic/repositories_str.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../models/repositories.dart';
+
+const String kEngineStr = 'engine';
```

Comment:
Avoid having types in the variable name. Would something like `kEngineRepo` make more sense?